bin/ci

```diff
@@ -73,27 +73,27 @@ prepare_system() {
 }
 
 build() {
-  with_build_env 'make std_spec clean threads=1'
+  with_build_env 'make std_spec clean threads=1 junit_output=.'
 
   case $ARCH in
     i386)
       with_build_env 'make crystal threads=1'
-      with_build_env 'SPEC_SPLIT="0%4" make std_spec threads=1'
-      with_build_env 'SPEC_SPLIT="1%4" make std_spec threads=1'
-      with_build_env 'SPEC_SPLIT="2%4" make std_spec threads=1'
-      with_build_env 'SPEC_SPLIT="3%4" make std_spec threads=1'
+      with_build_env 'SPEC_SPLIT="0%4" make std_spec threads=1 junit_output=.'
+      with_build_env 'SPEC_SPLIT="1%4" make std_spec threads=1 junit_output=.'
+      with_build_env 'SPEC_SPLIT="2%4" make std_spec threads=1 junit_output=.'
+      with_build_env 'SPEC_SPLIT="3%4" make std_spec threads=1 junit_output=.'
 
       parts=16
       i=0
       while [ $i -lt $parts ]; do
-        with_build_env "CRYSTAL_SPEC_COMPILER_THREADS=1 SPEC_SPLIT=\"$i%$parts\" make compiler_spec threads=1"
+        with_build_env "CRYSTAL_SPEC_COMPILER_THREADS=1 SPEC_SPLIT=\"$i%$parts\" make compiler_spec threads=1 junit_output=."
         i=$((i + 1))
       done
 
       with_build_env 'make docs threads=1'
       ;;
     *)
-      with_build_env 'make crystal std_spec compiler_spec docs threads=1'
+      with_build_env 'make crystal std_spec compiler_spec docs threads=1 junit_output=.'
       ;;
   esac
```

Comment:
This needs to be changed in order to harvest junit output files from all of the runs.
Sth TBD.

Reply:
Done (https://circleci.com/gh/crystal-lang/crystal/35680)

![image](https://user-images.githubusercontent.com/988/71429794-c47c0c00-26c8-11ea-997c-6600df897182.png)